zips/zip-0226.rst

```diff
@@ -481,20 +481,55 @@ The details of the computation of this value are in ZIP 227 [#zip-0227-txiddiges
 Signature Digest 
 ================
 
-The details of the changes to this algorithm are in ZIP 227 [#zip-0227-sigdigest]_.
+The per-input transaction digest algorithm to generate the signature digest in ZIP 244 [#zip-0244-sigdigest]_ is modified so that a signature digest is produced for each transparent input, each Sapling input, each OrchardZSA Action, and additionally for each Issuance Action.
+The modifications replace the ``orchard_digest`` in ZIP 244 with a new ``orchard_zsa_digest``, and add a new branch, ``issuance_digest``, for the Issuance Action information.
+
+The overall structure of the hash is as follows. We highlight the changes for the OrchardZSA protocol via the ``[ADDED FOR ZSA]`` text label, and we omit the descriptions of the sections that do not change for the OrchardZSA protocol::
+
+    signature_digest
+    ├── header_digest
+    ├── transparent_sig_digest
+    ├── sapling_digest
+    ├── orchard_zsa_digest      [ADDED FOR ZSA]
+    └── issuance_digest         [ADDED FOR ZSA]
+
+signature_digest
+----------------
+A BLAKE2b-256 hash of the following values ::
+
+   S.1: header_digest          (32-byte hash output)
+   S.2: transparent_sig_digest (32-byte hash output)
+   S.3: sapling_digest         (32-byte hash output)
+   S.4: orchard_zsa_digest     (32-byte hash output)  [ADDED FOR ZSA]
+   S.5: issuance_digest        (32-byte hash output)  [ADDED FOR ZSA]
+
+The personalization field remains the same as in ZIP 244 [#zip-0244]_, namely::
+
+  "ZcashTxHash_" || CONSENSUS_BRANCH_ID
```

Comment:
Can you elaborate on this?
I don't remember seeing such thing anywere in the code.

Reply:
It is in `librustzcash`: See [here](https://github.com/QED-it/librustzcash/blob/5ea95ea2bfa55ccd8d39c1be39eb4978ec338045/zcash_primitives/src/transaction/txid.rs#L411).

Reply:
To elaborate: We didn't expand on this in the ZIP since originally this top level branch is the same as in ZIP 244, we had only added a new sub branch for issuance. Now we are expanding it a bit because we are replacing `orchard_digest` with `orchard_zsa_digest` and adding the `issuance_digest` branch. 

Implementation wise as well, the personalization of `"ZcashTxHash_" || CONSENSUS_BRANCH_ID` wasn't actively changed (and still isn't).

Reply:
The alternative is to just mention it is unchanged from ZIP 244, without mentioning what it is